### PR TITLE
Bump in Harmonic: msgs and transport

### PR DESCRIPTION
**Pull request by caguero**

### CODE REVIEW

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 1]
```diff
@@ -27,11 +27,11 @@ gz_collections = [
           'math'      : [ debbuild: 'gz-math7'       , branch: 'gz-math7' ],
           'plugin'    : [ debbuild: 'gz-plugin2'     , branch: 'gz-plugin2' ],
           'common'    : [ debbuild: 'gz-common5'     , branch: 'gz-common5' ],
-          'msgs'      : [ debbuild: 'gz-msgs9'       , branch: 'gz-msgs9' ],
+          'msgs'      : [ debbuild: 'gz-msgs9'       , branch: 'main' ],
```
Comment (j-rivero) [inline jenkins-scripts/dsl/ignition_collection.dsl:30]:
```suggestion
          'msgs'      : [ debbuild: 'gz-msgs10'       , branch: 'main' ],
```

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 2]
```diff
@@ -27,11 +27,11 @@ gz_collections = [
...
           'rendering' : [ debbuild: 'gz-rendering8'  , branch: 'main' ],
           'sdformat'  : [ debbuild: 'sdformat13'     , branch: 'sdf13' ],
           'fuel-tools': [ debbuild: 'gz-fuel-tools8' , branch: 'gz-fuel-tools8' ],
-          'transport' : [ debbuild: 'gz-transport12' , branch: 'gz-transport12' ],
+          'transport' : [ debbuild: 'gz-transport12' , branch: 'main' ],
```
Comment (j-rivero) [inline jenkins-scripts/dsl/ignition_collection.dsl:34]:
```suggestion
          'transport' : [ debbuild: 'gz-transport13' , branch: 'main' ],
```

**Comment by caguero** [comment]
Not sure about this CI failure. @j-rivero , is this expected?

**Review by j-rivero — COMMENTED** [review]
Need to bump the debbuild job name.

**Comment by j-rivero** [comment]
> Not sure about this CI failure. @j-rivero , is this expected?

Seems unrelated to the change, somehow the download of the jar file is corrupted. Ignore it by now.

**Pull request merged**